Proposed/asc049.md

```diff
@@ -2,21 +2,20 @@
 Add `concentration` Unit to `Units` Definitions
 
 #### **Authorship**
-Alex Hadik alex@transcriptic.com
+Alex Hadik alex.hadik@strateos.com
 
 #### **Motivation**
 We should formalize how we represent concentration so that we have a standard way of defining solution concentrations.
 
 #### **Proposal**
-Add an arbitrary `item` "unit" to the `units` definition list. This is used as a catch all for discrete items such as cells, beads, etc.
 
-Add three types of concentration units to the `units` definition list: mass, molarity and arbitrary item. These concentration types will be defined as follows:
+Add three types of concentration units to the `units` definition list: mass, volume and molarity. These concentration types will be defined as follows:
 
 - MassConcentration: `milligram/milliliter`, `nanogram/microliter`
+- VolumeConcentration: `milliliter/milliliter`, `microliter/microliter`
 - MolarConcentration: `mole/liter`, `millimole/liter`, `micromole/liter`
-- ItemConcentration: `item/liter`, `item/milliliter`, `item/microliter`
 
-These units are validated by validating the numerator and denominator units. The numerator must be any valid unit from the following set: (`Mass`, `Matter`, `Volume`, `Item`) The denominator must be a recognized unit from the `Volume` category.
+These units are validated by validating the numerator and denominator units. The numerator must be any valid unit from the following set: (`Mass`, `Matter`, `Volume`) The denominator must be a recognized unit from the `Volume` category.
```

Comment:
Maybe `Amount` or `Molar Amount` instead of `Matter`?

Reply:
we currently use `Matter` as part of our spec, which we could certainly update separately